baselines/webworker.generated.d.ts

```diff
@@ -2318,10 +2330,6 @@ declare var NavigationPreloadManager: {
     new(): NavigationPreloadManager;
 };
 
-interface NavigatorBeacon {
-    sendBeacon(url: string, data?: Blob | Int8Array | Int16Array | Int32Array | Uint8Array | Uint16Array | Uint32Array | Uint8ClampedArray | Float32Array | Float64Array | DataView | ArrayBuffer | FormData | string | null): boolean;
-}
-
```

Comment:
We're a tad worried about this getting removed, looks like it's being used still: https://caniuse.com/#search=Beacon

Probably just needs: `https://www.w3.org/TR/beacon/` being added as an IDL source